app/views/users/two_factor_authentication/max_login_attempts_reached.html.erb

```diff
@@ -0,0 +1,12 @@
+<% content_for :page_title, "You've reached the maximum amount of sign in attempts" %>
```

Comment:
This new page looks fine, but it might be worth mentioning in the commit message that it's been added.

Reply:
Beth suggested we don't focus on that new journey because it's undergoing substantial changes, hence why it's been left out. You can turn off the new journey by going in `config/development.rb` and editing for this: 

```
  config.enable_enhanced_2fa_experience = false
```

Note that production is not running the new journey either.